ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/ApiServices.scala

```diff
@@ -71,9 +72,9 @@ object ApiServices {
       commandConfig: CommandConfiguration,
       optTimeServiceBackend: Option[TimeServiceBackend],
       loggerFactory: NamedLoggerFactory,
-      metrics: MetricRegistry)(
-      implicit mat: ActorMaterializer,
-      esf: ExecutionSequencerFactory): Future[ApiServices] = {
+      metrics: MetricRegistry,
+      healthChecks: HealthChecks,
+  )(implicit mat: ActorMaterializer, esf: ExecutionSequencerFactory): Future[ApiServices] = {
```

Comment:
Not your fault but I'd say it's time to maybe thing about ways in which we could refactor this class. Lots of concerns tied up in a single place. :thinking:

Reply:
Agreed. I'll add it to the list of stuff to do after this is merged.

Reply:
FYI @gerolf-da and I had a quick chat about packaging together given concerns together to make the dependency hierarchy a bit taller but less broad, let's talk about how we want to proceed.